dsa-solutions/lc-solutions/1300-1399/1305- All- Elements- in- Two- Binary -Search -Trees.md

```diff
@@ -0,0 +1,164 @@
+---
+id: All- Elements- in- Two- Binary -Search -Trees
```

Comment:
replace `All- Elements- in- Two- Binary -Search -Trees` to `all-elements-in-two-binary-search-trees`